warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__shapes_for_all_trips.sql

```diff
@@ -0,0 +1,49 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__schedule_feed_guideline_index') }}
+),
+
+dim_trips AS (
+    SELECT * FROM {{ ref('dim_trips') }}
+),
+
+summarize_trips AS (
+   SELECT
+       feed_key,
+       EXTRACT(date FROM _valid_from) AS valid_from,
+       EXTRACT(date FROM _valid_to) AS valid_to,
+       COUNTIF(shape_id IS NOT NULL) AS ct_shape_trips,
+       COUNT(*) AS ct_trips
+    FROM dim_trips
+   GROUP BY 1, 2, 3
+),
+
+daily_trips AS (
+  SELECT
+    t1.date,
+    t1.feed_key,
+    SUM(t2.ct_shape_trips) AS tot_shape_trips,
+    SUM(t2.ct_trips) AS tot_trips
+  FROM feed_guideline_index AS t1
+  LEFT JOIN summarize_trips AS t2
+       ON t1.date >= t2.valid_from
+       AND t1.date <= t2.valid_to
```

Comment:
Wondering if we want this to be a strict inequality, I think that's what we've generally done in v1

Reply:
i was hoping you'd look at this - to me it's a question of how valid_to is designed. totally your call!

Reply:
Oh, I was forgetting how the changes we made are supposed to work. I think the new guidance would just be to aggregate to the `feed_key` level and then left join onto the date index and ditch the date logic in the join entirely. Because we've done the ranged joins to construct the feed check date index already. (`valid_to` is now a timestamp and it's no longer a date-level operation, which is why the `feed_daily` table checks for what was in effect at noon Pacific and then calls that the feed for the whole day.)

Reply:
I hope that makes sense

Reply:
I think so! I'll just keep the join as-is, except that I'll remove date constraints entirely. Basically, feed_key gets me the date info I need. Is that right?

Reply:
Yes. I am sorry for writing a paragraph without that as the exact TLDR 😅 

Also means I think you can drop the valid to and from columns from the select above but that is less important